back-office/customer-service/customer-service/dashboard.md

```diff
@@ -0,0 +1,304 @@
+# SPECIFICATIONS - CUSTOMERS SERVICE > CUSTOMERS SERVICE PAGE
+
+## MOCKUPS
+https://projects.invisionapp.com/d/main?origin=v7#/console/20267309/425376946/preview?scrollOffset=0
+https://projects.invisionapp.com/d/main?origin=v7#/console/20267309/425548149/preview?scrollOffset=0
+https://projects.invisionapp.com/d/main?origin=v7#/console/20267309/425548150/preview?scrollOffset=0
+https://projects.invisionapp.com/d/main?origin=v7#/console/20267309/425549455/preview?scrollOffset=0
+
+## STATS
+
+There are 3 indicators displayed:
+
+- Pending discussion threads
+- Average response time for 30 days in hours
+- Message per thread for 30 days
+
+Theses indicators can be refreshed by clicking on the refresh icon.
+
+## CONTACTS BLOCKS
+
+Each contact added in in Shop parameters > Contact > Contact has its own block.
+
+With these blocks, the users can quickly see if a there are new messages (that have not yet been read). 
+
+By default, there are two blocks, and adding more contacts will add more blocks.
+
+For each block, the description filled for the contact (in Shop parameters > Contact > Contact) is displayed.
+
+The number of **opened** messages for the contact are displayed: "X new message(s)" with a link opening the first message.
+
+If there is no message, then "No new messages" is displayed.
+
+## MEANING OF STATUS
+
+A simple reminder of the 4 status that can be applied to a discussion thread:
+
+- Open
+- Closed
+- Pending 1
+- Pending 2
+
+## STATISTICS
+
+An overview of the global activity of the customer service since the beginning with 5 indicators displayed:
+
+- Total threads 
+- Threads pending 
+- Total number of customer messages 
+- Unread threads 
+- Closed threads 
+
+## CUSTOMER SERVICE
+
+### Settings wheel
+
+When clicking on it, two options are displayed:
+
+1) Show SQL query
+
+After clicking on it, the sql query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+2) Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manage
+
+### Bulk actions drop-down
+
+This dropdown is enabled only if at least one message is selected, otherwise it's disabled.
+
+There is 5 options:
+
+1) Mark as "opened"
+
+After clicking on it, the status of the selected messages are changed for "opened" and a successfull message is displayed "The status of the selection has been successfully changed."
```

Comment:
@Julievrz this successfull message is new, could you please verify the it ? :)

Reply:
Hi @marionf, there is a string that already exists: "The status has been successfully updated." in Admin.Notifications.Success 
Couldn't we keep this one? Or is this wording going to apply only when several messages are selected?

Reply:
This message will be displayed only when several messages are selected.
Maybe we can simply use the existing wording: "Successful update.", wdyt ?

Reply:
I think we can use a more general wording, yes. We tend to prefer "Update successful" 😊 (without punctuation)

Reply:
Ok, so let's go for this one, thank you @Julievrz !